test/runLint

```diff
@@ -52,6 +52,9 @@ gitgrep '(?<!command)'"$cmdstart"'(grep|ls|sed|cd)(\s|$)' \
 gitgrep '(?<!command)'"$cmdstart"'awk(\s|$)' \
     'invoke awk through "_comp_awk"'
 
+gitgrep '@\([^()|$]+\)' \
```

Comment:
The commit that adds this has a "unnecesasry" typo in the commit message. Could fix up before merging.

Reply:
Thanks. I fixed this in the latest push too.